tests/torch/test_sota_checkpoints.py

```diff
@@ -9,681 +9,669 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import csv
 import datetime
 import json
 import os
-import re
-import shlex
-import subprocess
 import sys
 from collections import OrderedDict
+from dataclasses import dataclass
 from pathlib import Path
 from typing import List, Optional, Tuple
 
+import pandas as pd
 import pytest
-from prettytable import PrettyTable
-from yattag import Doc
 
-from nncf.common.utils.os import is_linux
-from nncf.common.utils.os import is_windows
-from nncf.config import NNCFConfig
 from tests.shared.metric_thresholds import DIFF_FP32_MAX_GLOBAL
 from tests.shared.metric_thresholds import DIFF_FP32_MIN_GLOBAL
-from tests.shared.metric_thresholds import DIFF_TARGET_MAX_GLOBAL
-from tests.shared.metric_thresholds import DIFF_TARGET_MIN_GLOBAL
 from tests.shared.paths import DATASET_DEFINITIONS_PATH
 from tests.shared.paths import PROJECT_ROOT
 from tests.shared.paths import TEST_ROOT
+from tests.torch.helpers import Command
 
-BG_COLOR_GREEN_HEX = "ccffcc"
-BG_COLOR_YELLOW_HEX = "ffffcc"
-BG_COLOR_RED_HEX = "ffcccc"
+DIFF_TARGET_PT_MIN = -0.1
+DIFF_TARGET_PT_MAX = 0.1
+DIFF_TARGET_OV_MIN = -0.01
+DIFF_TARGET_OV_MAX = 0.01
```

Comment:
What is the `minimum` difference? Why is the target for PyTorch and OpenVINO 10 times differ?

Reply:
> What is the minimum difference?
https://github.com/openvinotoolkit/nncf/blob/08cd0f7de2de04d0cdaf1c8b326ad96323c342af/tests/torch/test_sota_checkpoints.py#L314

>Why is the target for PyTorch and OpenVINO 10 times differ?

Because metrics measured on PT are unstable.

Reply:
> Because metrics measured on PT are unstable.

In my opinion, it's not the reason to roughly extend the threshold for all models 10 times. Also, there was no such threshold in the old version of the test.
In any case, we should decide this with the thoughts of @alexsu52 and @vshampor.

Reply:
> In my opinion, it's not the reason to roughly extend the threshold for all models 10 times. 

I did not extend thresholds for all models, by default thresholds was +-0.1, now this thresholds keep for PT and reduced for OV.


> Also, there was no such threshold in the old version of the test.
It was.
 https://github.com/openvinotoolkit/nncf/blob/08cd0f7de2de04d0cdaf1c8b326ad96323c342af/tests/torch/test_sota_checkpoints.py#L304-L314